app/components/results-list/component.js

```diff
@@ -4,11 +4,19 @@ import Ember from 'ember';
 export default Ember.Component.extend({
 
     data : [],
+    pagebackbtn : null,
+    pagenextbtn : null,
 
     init(){
         this._super(...arguments);
         let data = this.processData(this.get('data.hits.hits'));
         this.set('data', data);
+       
```

Comment:
What happens here if the component doesn't get removed or added to the page, but the page changes? (this is an improvement that is planned)